tests/prefetch_related/models.py

```diff
@@ -244,7 +244,7 @@ def __str__(self):
         return "%s (%s)" % (self.name, self.id)
 
 
-## Ticket #21410: Regression when related_name="+"
+# --- Ticket #21410: Regression when related_name="+" ---
```

Comment:
I'm fine if you want to use this style in GIS, but more the rest of the code, I think a single # is sufficient.